tests/setuponlytests/conflicting_world_for_spigot_server/verify.sh

```diff
@@ -0,0 +1,5 @@
+mc-image-helper assert fileExists world/level.dat && \
```

Comment:
Excited to see these, will need to go back and add for forge api and vanilla tweaks.

Reply:
Agreed, I'm so happy to see others already using the new assertions I just added